src/Components/Category.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {CategoryProps} from '../Types';
+import {Img, staticFile} from 'remotion';
+
+export const Category: React.FC<CategoryProps> = ({
```

Comment:
Maybe rename this with `IconWithText` or something more representative that `Category` ?